lambdatimes/src/components/TopBar.js

```diff
@@ -1,23 +1,100 @@
 import React from 'react';
+import styled from 'styled-components';
 
 // Refactor this component to use styled components and not classNames. 
 // You can find the corresponding CSS in the CSS/index.css file
+export const TopBarStyled = styled.div`
```

Comment:
You can split these into smaller styled components so that the styles are more reusable. 👍